pangeo_forge_esgf/params.py

```diff
@@ -0,0 +1,27 @@
+# dataset id templates
+cmip6_template = "mip_era.activity_id.institution_id.source_id.experiment_id.variant_label.table_id.variable_id.grid_label.version"
+cordex_template = "project.product.domain.institute.driving_model.experiment.ensemble.rcm_name.rcm_version.time_frequency.variable"
+
+# request params
+cmip6_params = {
+    "type": "Dataset",
+    "retracted": "false",
+    "format": "application/solr+json",
+    "fields": "instance_id",
+    "latest": "true",
+    "distrib": "true",
+    "limit": 500,
+}
+
+cordex_params = {
+    "type": "Dataset",
+    "format": "application/solr+json",
+    "fields": "instance_id",
+    "latest": "true",
+    "distrib": "true",
+    "limit": 500,
+}
```

Comment:
Lots of repeated code here. As far as I can see the only different is `"retracted": "false"`? Could you refactor this to a 'base_params` dict, that is then updated for each project?

Reply:
Yes, absolutely, no problem!